content/javascript/concepts/dates/terms/toDateString/toDateString.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'toDateString()'
+Description: 'returns a string representing the date portion of a Date instance' 
+Subjects: 
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`toDateString()`** method of Date instances returns the date portion of a Date object as a string.
```

Comment:
Make this a bit more descriptive.

Reply:
Hi, thanks for the feedback. I'm not sure I understand what you mean by "more descriptive". Is there a specific way you would like me describe the method?

Reply:
@yeipkwa sure. So more descriptively, I wanted you to add more info about `toDateString()`. For example: When applied to a Date object, `toDateString()` retrieves the date part of the object's value and formats it as a string representing the date in a locale-specific format. We can add this sentence to this sentence too.

Reply:
Apart from this comment please check out the other comments and work on them too. 🚀

Reply:
Okay, got it. Thank you!